static/css/survey-notice.css

```diff
@@ -0,0 +1,6 @@
+.survey-notice {
+    padding: 20px;
+    margin: 20px 0;
+    border-radius: 3px;
+    background-color: #eeeeee;
```

Comment:
I recommend setting `color` as well. Maybe that's outdated advice but it was something I remember from CSS1.

Reply:
Kinda funny - I copied that code verbatim from deprecation-warning.css